src/main/java/nl/novi/techiteasycontroller/controllers/RemoteControllerController.java

```diff
@@ -0,0 +1,61 @@
+package nl.novi.techiteasycontroller.controllers;
+
+import nl.novi.techiteasycontroller.dtos.RemoteControllerOutputDto;
+import nl.novi.techiteasycontroller.repositories.TelevisionRepository;
+import nl.novi.techiteasycontroller.service.RemoteControllerService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class RemoteControllerController {
+
+    private final RemoteControllerService remoteControllerService;
+
+    public RemoteControllerController(RemoteControllerService remoteControllerService) {
+        this.remoteControllerService = remoteControllerService;
+    }
+
+    @GetMapping("/remotecontrollers")
+    public ResponseEntity <List<RemoteControllerOutputDto>> getAllRemoteControllers() {
+        return ResponseEntity.ok(this.remoteControllerService.getAllRemoteControllers());
```

Comment:
Je vergeet hier nog een list te maken waarin je de remote controllers opslaat. die lijst kan je dan returnen

Reply:
Dat is niet altijd nodig. In dit geval gebruikt Ayoub direct de List<dto> output van de service methode als body van de ResponseEntity